src/js/pieChart.js

```diff
@@ -75,9 +75,16 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             // outerRadius: number,
             // innerRadius: number,
             // animationDuration: number,
-            // sort: boolean   // Whether or not to sort the data by values when creating the legend
-            // pieTitle: the accessible title to be applied to the pie chart
-            // pieDescription: the accessible description to be applied to the pie chart
+            // sort: boolean   // Whether or not to sort the data by values when creating the legend,
+            // pieTitle: the accessible title to be applied to the pie chart,
```

Comment:
To be consistent with comments above, add the variable type, such as: 
```
pieTitle: string,    // the accessible title to be applied to the pie chart
```
Applies to line 79-87.